docs/using-profiles.md

```diff
@@ -0,0 +1,43 @@
+(using-lint-profiles)=
+
+# Applying profiles
+
+Ansible-lint profiles allow content creators to progressively improve the quality of Ansible playbooks, roles, and collections.
+
+During early development cycles, you need Ansible-lint rules to be less strict.
+Starting with the minimal profile ensures that Ansible can load your content.
+As you move to the next stage of developing content, you can gradually apply profiles to avoid common pitfalls and brittle complexity.
+Then, when you are ready to publish or share your content, you can use the `shared` and `production` profiles with much stricter rules that harden security and guarantee reliability.
```

Comment:
and guarantee reliability.  The `shared` and `production` profiles exist to ensure your Ansible content is easy to read and consistently formatted, making it easier for other to contribute to it and use. ( <=something like this? Trying to reinforce why these profiles exist)

Reply:
@cidrblock "These profiles harden security, guarantee reliability, and ensure your Ansible content is easy for others to contribute to and use."

Does that sound reasonable or should I drop the "harden security" and "guarantee reliability"?